xbmc/GUIInfoManager.cpp

```diff
@@ -9451,18 +9432,13 @@ bool CGUIInfoManager::GetItemInt(int &value, const CGUIListItem *item, int info)
       if (item->IsFileItem())
       {
         const CFileItem *pItem = static_cast<const CFileItem*>(item);
-        if (pItem && pItem->HasPVRChannelInfoTag())
-        {
-          CPVREpgInfoTagPtr epgNow(pItem->GetPVRChannelInfoTag()->GetEPGNow());
-          if (epgNow)
-            value = (int) epgNow->ProgressPercentage();
-        }
-        else if (pItem && pItem->HasEPGInfoTag())
+        if (pItem->IsPVRChannel() || pItem->IsEPG())
```

Comment:
There used to be a null check here, did you remove it intentionally?

Reply:
Yes, this is by intention. If `item->IsFileItem()` returns `true`, the cast `const CFileItem *pItem = static_cast<const CFileItem*>(item);` must succeed.

Reply:
Cool